### DISCUSSION TITLE
essay/2023/07/28/coding-in-korean

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#185] [Comments]
# essay/2023/07/28/coding-in-korean

한글로 코딩해도 괜찮다고 생각합니다.

https://jeho.page/essay/2023/07/28/coding-in-korean.html

**Comment by voidFunctionReturn0** [comment]
오, 미처 생각해보지 못햇는데 넘 좋은 아이디어네요 ㅎㅎ
토스 글도 많은 참고가 될 것 같습니다

변수, 함수 이름까지는 몰라도, enum을 한글로 쓰는 것은 매우 편할 것 같아요!
